flax/nnx/bridge/module.py

```diff
@@ -183,8 +180,9 @@ def _module_meta_call(cls: type[M], *args, **kwargs) -> M:
 
 class AttrPriority(enum.IntEnum):
   HIGH = 1
-  DEFAULT = 2
-  LOW = 3
+  INIT_PARENT = 2
+  DEFAULT = 3
+  LOW = 4
```

Comment:
What if we leave some space between them as a escape hatch in case more control is needed?

```python
HIGH = 0
INIT_PARENT = 20
DEFAULT = 50
LOW = 100
```

Reply:
Done.